### ISSUE TITLE
is `IdentityCredentialRequestOptionsContext` for FedCM UI only?

### ISSUE THREAD

**Issue by panva** [fedidcg/FedCM#603]
This `context`, which the RP can select one of a predefined list, controls the wording in the browser UI.

It would be helpful to get this RP-selected context in the login_url as well as at the assertion endpoint to discern between the language used on the login page as well as in the possible Continuation API triggered popup that renders IdP-provided UI.

**Comment by npm1** [comment]
It is for FedCM UI only, yea. It seems plausible to have it in some of those, if you have specific use cases in mind?

**Comment by samuelgoto** [comment]
Yeah, this seems like a somewhat trivial and harmless (in theory, it increases a bit of entropy in the `login_url`, but I think it is plausible) change to make.
